DirectEmployers/MyJobs
# [PD 3114] remove beta feedback link

## What's this PR do?
This PR removes the Leave Feedback link from the Analytics sidebar

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the Analytics App

## How should this be manually tested?
Pull down the repository. When you go into the Analytics application you will see that the Leave Feedback has been removed from the left hand sidebar

## Any background context you want to provide?
